tests/envalid.ts

```diff
@@ -32,20 +36,71 @@ const spec: Specs = {
         choices: [1, 2, 3]
     }),
     json: json({
-        devDefault: { foo: 'bar' },
+        devDefault: { foo: 'bar' }
     }),
     url: url(),
     email: email({
         example: 'example',
         docs: 'http://example.com'
     })
 }
-spec[0]._parse('test');
-spec[0].type === 'test';
```

Comment:
Not too sure what `spec[0]` was supposed to be doing here -- this isn't valid for an object that doesn't have that property, but `spec` being declared as type `Specs` was causing it to pass the typechecker. There's not really any reason to have a `Specs` interface, either, so I've removed it.

Reply:
Sounds good. I have no idea what `spec[0]` was doing there 😆